tests/helperDb.js

```diff
@@ -175,13 +175,7 @@ module.exports.createTemplate = async function createTemplate() {
   });
 };
 
-/**
- * @this {import('mocha').Context}
- */
 module.exports.dropTemplate = async function dropTemplate() {
-  // long timeout because DROP DATABASE might take a long time to error
-  // if other processes have an open connection to that database
-  this.timeout?.(20000);
```

Comment:
I pushed this up into the function that actually calls this one; that seems better than having Mocha-specific stuff in here? I had to do that because TypeScript started complaining about `this` being the incorrect type at the callsite of this function.

Reply:
Yep, seems reasonable.

Reply:
As it turns out, `this.timeout` is straight up not defined in global fixtures! I just removed the timeout entirely.